### PR TITLE
CSHARP-754 DataStax Apollo support

### PR DESCRIPTION
**Pull request by joao-r-reis**

### CODE REVIEW

File: build.yaml [hunk 1]
```diff
@@ -45,7 +45,19 @@ build:
 
       echo "========== Copying ssl files to $HOME/ssl =========="
       cp -r /home/jenkins/ccm/ssl $HOME/ssl
-
+      
+      export SNI_ENABLED="true"
```
Comment (jorgebay) [inline build.yaml]: NIT: given that SNI project uses a single DSE version, we could build it on a single dimension of the matrix, e.g., build it only on `'3.11'` jobs

Comment (joao-r-reis) [reply build.yaml]: done, already checked in jenkins that it is correctly skipping the cloud tests for targets other than `3.11` do you agree with the way I implemented this in `build.yaml`? It's not very pretty but I didn't know any other way to do this

File: .travis.yml [hunk 2]
```diff
@@ -2,23 +2,23 @@ language: csharp
...
     - dse_3.1
-    
-env:
-  global:
-    - BuildCoreOnly=True
-  matrix:
-    - TARGETF="netcoreapp2.1" BUILD_EXAMPLES="1"
-    - TARGETF="netcoreapp2.0" BUILD_EXAMPLES="0"
+
+matrix:
```
[the remark this answers is not in the document]
Comment (joao-r-reis) [reply .travis.yml]: Nice! I didn't know about that, I'll look into it after this release

File: build.yaml (hunk 1 above)
Comment (jorgebay) [reply build.yaml]: yeah, looks good. We're already overriding `CASSANDRA_VERSION` so makes a code a little more complicated but its OK :)

**Pull request merged**